src/main/java/controller/NewProductServlet.java

```diff
@@ -0,0 +1,36 @@
+package controller;
+
+import factory.ProductServiceFactory;
+import model.Product;
+import service.impl.ProductServiceImpl;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet(value = "/newProduct")
+public class NewProductServlet extends HttpServlet {
+
+  private static final ProductServiceImpl PRODUCT_SERVICE = ProductServiceFactory.getInstance();
```

Comment:
sorry) change to previous version, because this field is mutable so it isn't constant
```suggestion
  private static final ProductServiceImpl productService = ProductServiceFactory.getInstance();
```